string human_detection_start
string face_recognition_start
string action_recognition_start